### ISSUE TITLE
Timeout in degree 3 Maass form browse page

### ISSUE THREAD

**Issue by roed314** [LMFDB/lmfdb#5846]
We very frequently get the following error in the flasklog on www.lmfdb.org:

```
Exception on /L/degree3/MaassForm/ [GET]
Traceback (most recent call last):
  File "/home/sage/sage-10.1/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sage/sage-10.1/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/flask/app.py", line 1519, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sage/sage-10.1/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/flask/app.py", line 1517, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sage/sage-10.1/local/var/lib/sage/venv-python3.10/lib/python3.10/site-packages/flask/app.py", line 1503, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/lmfdb/lmfdb-git-web/lmfdb/lfunctions/main.py", line 840, in l_function_maass_gln_browse_page
    contents = LfunctionPlot.getAllMaassGraphHtml(degree)
  File "/home/lmfdb/lmfdb-git-web/lmfdb/lfunctions/LfunctionPlot.py", line 66, in getAllMaassGraphHtml
    ans += getOneGraphHtml([g, l])
  File "/home/lmfdb/lmfdb-git-web/lmfdb/lfunctions/LfunctionPlot.py", line 185, in getOneGraphHtml
    graphInfo = getGraphInfo(gls)
  File "/home/lmfdb/lmfdb-git-web/lmfdb/lfunctions/LfunctionPlot.py", line 203, in getGraphInfo
    (width, height) = getWidthAndHeight(gls)
  File "/home/lmfdb/lmfdb-git-web/lmfdb/lfunctions/LfunctionPlot.py", line 234, in getWidthAndHeight
    for origin in db.lfunc_lfunctions.search({'group': gls[0], 'conductor': gls[1]}, 'origin'):
  File "/home/lmfdb/lmfdb-git-web/lmfdb/backend/searchtable.py", line 541, in _search_iterator
    for rec in cur:
psycopg2.errors.QueryCanceled: canceling statement due to statement timeout

[2024-02-02 14:15:46 UTC] 500 error on URL https://www.lmfdb.org/L/degree3/MaassForm/ ()
```
It would be good to change the code so that it's not trying to search on `lfunc_lfunctions`, either by caching the result or at least switching to `lfunc_search`, which should be more optimized.

**Comment by edgarcosta** [comment]
This doesn't even look okay to me: https://beta.lmfdb.org/L/degree3/MaassForm/

Should we redirect to: https://www.lmfdb.org/L/?degree=3&origin=MaassGL3 ?

**Comment by AndrewVSutherland** [comment]
I'd like to know where this link is coming from (or is just a bot checking an old link?)

**Comment by davidfarmer** [comment]
I noticed that the SVG was not rendering, but I forgot to submit an issue.

I occasionally go directly to that page.

**Comment by davidfarmer** [comment]
On hold until #6236

**Comment by roed314** [comment]
@davidfarmer, should this still wait on #6236?

**Comment by davidfarmer** [comment]
That page should be trashed.   That is, redirect as @edgarcosta suggested.

 When we actually get to #6236, this page will be redone from scratch,
and surely it should also be at a different URL.
